b3lb/rest/templates/getRecordings.xml

```diff
@@ -0,0 +1,44 @@
+<response>
+    <returncode>SUCCESS</returncode>
+    <recordings>{% for record in records %}
+        <recording>
+            <recordID>{{ record.uuid }}</recordID>
+            <meetingID>{{ record.meeting_id }}</meetingID>
+            <internalMeetingID>{{ record.internal_meeting_id }}</internalMeetingID>
+            <name>{{ record.name }}</name>
```

Comment:
Are all strings in this template already escaped or is it sure that they are XML safe? Maybe use Jinja2's [escape](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.escape) method?